src/components/dropdown/index.js

```diff
@@ -17,24 +17,85 @@ export default class Dropdown extends Component {
     }
 
     render() {
+
         let {props, state} = this
 
-        return (
-            <div className={"boostrap-dropdown " + style.dropdown}>
-                <div className={"dropdown-toggle " + style.toggle} data-toggle="dropdown">
+        //Item template
+        const Item = (props) => (
+            <li {...props}
+                className={style.item}
+                style={{backgroundColor: props.config.type === 'color' ? props.value : null}}
+                onClick={this.onSelect.bind(this, props.value)}>
+                    {props.config.showText ? props.value : null}
+            </li>
+        )
+
+        //Label template
+        const Label = (props) => (
+            <div className={style.label} style={{backgroundColor: state.selected}}/>
+        )
+
+        // Action template
+        const Action = (props) => (
+            <div className={style.action}>{props.value}</div>
+        )
```

Comment:
for example the items in `dropdownColor` are well the colors and the `action` is -> `getMoreColors`